docs/analytics/jenkins_jobs.md

```diff
@@ -117,6 +117,20 @@ analytics-configuration/automation/run-automated-task.sh AnswerDistributionWorkf
 
 In edx docs: [Enrollment](http://edx-analytics-pipeline-reference.readthedocs.io/en/latest/running_tasks.html#enrollment)
 
+Before running the daily incremental enrollment task, below, you should first run the history task that loads historical enrollment events.
```

Comment:
Could you please put this under a separate heading, like the [Module Engagement -- bootstrap task](https://github.com/dcadams/openedx-deployment/blob/548dce289ffb8616ec6c41e94476038da5cdbea4/docs/analytics/jenkins_jobs.md#module-engagement---bootstrap-task)

Reply:
And please include a link to the docs for the [history task](http://edx-analytics-pipeline-reference.readthedocs.io/en/latest/running_tasks.html#history-task).